Scripts/script-VectraSummary.yml

```diff
@@ -0,0 +1,44 @@
+commonfields:
+  id: VectraSummary
+  version: -1
+name: VectraSummary
+script: |+
+  import json
+
+  res = []
+  ctx = demisto.context()
+  summary, details = None, None
+  try:
+      summary = demisto.get(ctx, "Label/summary")
+      if summary:
+          summary = json.loads(ctx["Label/summary"])
+          if summary:
+              summary = {k: formatCell(summary[k]) for k in summary}
+
+      details = demisto.get(ctx, "Label/detection_detail_set")
+      if details:
+          details = json.loads(ctx["Label/detection_detail_set"])
+          if details:
+              details = {k: formatCell(details[0][k]) for k in details[0]}
+
+      bad_ips = None
+      if details:
+          bad_ips = details["dst_ip"]
+
+      demisto.setContext('bad_ips', bad_ips)
+      demisto.setContext('hashes', "")
+
+      res.append({'ContentsFormat': formats['table'], 'Type': entryTypes['note'], 'Contents': summary})
+      res.append({'ContentsFormat': formats['table'], 'Type': entryTypes['note'], 'Contents': details})
+  except Exception, ex:
+      demisto.results(res)
+
```

Comment:
What if no exception occurs? You still need to `demisto.results(res)`
And if an exception occurs it should return an error with the exception info, see here:
https://raw.githubusercontent.com/demisto/content/master/Scripts/script-ADIsUserMember.yml